contactsbackup/.gitignore

```diff
@@ -1 +1,4 @@
-/build
\ No newline at end of file
+// SPDX-FileCopyrightText: 2020, Torsten Grote <t@grobox.de>
+// SPDX-License-Identifier: CC0-1.0
```

Comment:
I am not sure a comment in this file works like this. Is it really necessary to add those to these small helper files?

Reply:
Indeed, files inside the `.gitignore` are excluded from the REUSE compliance testing.